tools/purge_duplicate_rubyrep_triggers.rb

```diff
@@ -16,6 +16,9 @@
 GROUP BY relname
 SQL
 
+REGION = File.exists?("./REGION") ? File.read("./REGION").strip : "0"
```

Comment:
We could do this without detecting the region by pulling the pending changes table name from the pg_class table:

`SELECT relname FROM pg_class WHERE relname LIKE 'rr%_pending_changes';`